air/Cargo.toml

```diff
@@ -64,10 +64,8 @@ pretty_assertions = "0.6.1"
 serde_json = "1.0.61"
 
 [features]
-default = ["rand"]
 check_signatures = []
 gen_signatures = []
-rand = ["air-interpreter-data/rand", "air-interpreter-signatures/rand"]
```

Comment:
I believe it will break `air-near-contract`.

Reply:
I added this line in some commits in this PR and then removed b/c it's doesn't influence RISC-0, though I think it's still necessary to add this with the `rand` feature bubbling in the air-interpreter-data crate, but in the next PR.

Reply:
So, in final version `air/Cargo.toml` file is untouched.

Reply:
Sorry for the fuss.